DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`]}`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|
+If a template has placeholder fields for personalised information such as name or application date, you need to provide their values in a map. For example:
 
-</details>
+```java
+Map<String, String> personalisation = new HashMap<>();
+personalisation.put("first_name", "Amala");
+personalisation.put("application_date", "2018-01-01");
+```
+If a template does not have any placeholder fields for personalised information, you must pass in an empty map or `null`.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>Click here for more information</summary>
+A unique identifier. This reference can identify a single unique notification or a batch of multiple notifications. If you do not have a reference, you must pass in an empty string or `null`.
 
-#### `emailAddress`
-The email address the email notification is sent to.
+```
+String reference='STRING';
+```
 
-#### `templateId`
+#### emailReplyToId (optional)
 
-The template id is visible on the template page in the application.
+This is an email reply-to address specified by you to receive replies from your users. Your service cannot go live until you set up at least one of these email addresses. To set up:
 
-#### `personalisation`
+1. Sign into your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you are not in the correct service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the Email section and select __Manage__ on the "Email reply to addresses" row.
+1. Select __Change__ to specify the email address to receive replies, and select __Save__.
 
-If a template has placeholders, you need to provide their values, for example:
+```
+String emailReplyToId='8e222534-7f05-4972-86e3-17c5d9f894e2'
+```
+
+If you do not have have an `emailReplyToId`, you must pass in an empty string or `null`.
+
+### Response
+
+If the request to the client is successful, you will receive the following `SendEmailResponse`:
 
 ```java
-Map<String, String> personalisation = new HashMap<>();
-personalisation.put("name", "Jo");
-personalisation.put("reference_number", "13566");
+UUID notificationId;
+Optional<String> reference;
+UUID templateId;
+int templateVersion;
+String templateUri;
+String body;
+String subject;
+Optional<String> fromEmail;
 ```
 
-#### `reference`
-An optional identifier you generate. The reference can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+### Error codes
+
+If the request is not successful, the client will raise an `HTTPError`:
```

Comment:
`HTTPError` is a `NotificationClientException`

Reply:
Changed